user-guide/Basic_Functionality/Trending/Advanced_analytics_trending/Working_with_trend_predictions.md

```diff
@@ -28,41 +27,63 @@ The predicted trend is displayed as a line surrounded by three shaded areas:
 
 Different types of trend prediction can be displayed, each in a different style.
 
 From DataMiner 10.0.12 onwards:
 
-- High-precision prediction is displayed as a dotted line.
+- **High-precision prediction** is displayed as a dotted line.
 
-- Short-term prediction is displayed with short dashes.
+  A high-precision prediction is generated by a model fit on the real-time trend data. This type of trend prediction offers the greatest level of detail but also has the shortest forecast time horizon.
 
-- Mid-term prediction is displayed with long dashes.
+- **Short-term prediction** is displayed with short dashes.
 
-- Long-term prediction is displayed with slightly longer dashes
+  A short-time prediction (prior to DataMiner 10.0.12: mid-term prediction) is generated by a model fit on the 5-minute averaged trend data. The use of 5-minute averages usually over extended time spans means that the forecast time horizon for a short-term prediction is typically longer compared to a high-precision prediction.
+
+- **Mid-term prediction** is displayed with long dashes.
+
+  A mid-term prediction is generated by a model fit on the 1-hour averaged trend data. This type of trend prediction offers a forecast time horizon that is typically longer compared to a short-term prediction.
+
+- **Long-term prediction** is displayed with slightly longer dashes.
+
+  A long-term prediction is generated by a model fit on an aggregated set of 1-hour trend data points. This type of trend prediction extrapolates the global trend in the data to the future.
 
 Prior to DataMiner 10.0.12:
 
-- Short-term prediction is displayed as a dotted line.
+- **Short-term prediction** is displayed as a dotted line.
+
+  A short-term prediction is generated by a model fit on the real-time trend data. This type of trend prediction offers the greatest level of detail but also has the shortest forecast time horizon.
+
+- **Mid-term prediction** is displayed with short dashes.
 
-- Mid-term prediction is displayed with short dashes.
+  A mid-term prediction is generated by a model fit on the 5-minute averaged trend data. The use of 5-minute averages usually over extended time spans means that the forecast time horizon for a mid-term prediction is typically longer compared to a short-term prediction.
 
-- Long-term prediction is displayed with long dashes.
+- **Long-term prediction** is displayed with long dashes.
 
-- Full-term prediction, which is based on either daily data points or a server-side aggregated set of one-hour data points, is also displayed with long dashes. (Available from DataMiner 10.0.11 onwards.)
+  A long-term prediction is generated by a model fit on the 1-hour averaged trend data. This type of trend prediction offers a forecast time horizon that is typically longer compared to a mid-term prediction.
+
+- **Full-term prediction** is also displayed with long dashes (Available from DataMiner 10.0.11 onwards).
```

Comment:
```suggestion
- **Full-term prediction** is also displayed with long dashes (Available from DataMiner 10.0.11 onwards).
  A full-term prediction is generated by a model fit on an aggregated set of 1-hour trend data points. This type of trend prediction extrapolates the global trend in the data to the future.
```
Am I correct to assume that the description you provided for a long-term prediction (from 10.0.12 onwards) above can also be entered here?

Reply:
yes, the "long-term prediction" of versions later than 10.0.12, used to be called "full-term prediction" in older versions. The description can be copied